docker-compose.yml

```diff
@@ -38,6 +44,7 @@ services:
     tty: true
     volumes:
       - data:/home/bpyuser/.data
+      - .env:/home/bpyuser/.env:ro
```

Comment:
seems a bit weird imo, i think better to map the env variables into the container using `env` so they're available from anything running inside it

Reply:
pretty sure bpy will not be happy with not having a .env file

Reply:
it'll be fine - it takes .env file values first, then environment variables 2nd

ideally everything should be in the environment of the running process "globally" rather than only in the file

Reply:
can merge for now anyways